plugins/scaffolder-backend/src/scaffolder/actions/builtin/github/helpers.ts

```diff
@@ -254,6 +258,51 @@ export async function createGithubRepoWithCollaboratorsAndTopics(
     }
   }
 
+  for (const key in environmentVariables) {
+    if ({}.hasOwnProperty.call(environmentVariables, key)) {
```

Comment:
```suggestion
    if (Object.hasOwnProperty.call(environmentVariables, key)) {
```

hehe, no need to change this but constructing a new object each time just feels wrong :)

Would also feel even better with `for (const [key, value] of Object.entries(environmentVariables)) {...`